### PR TITLE
Setup environment

### PR DESCRIPTION
**Pull request by JasonYe007**
#patch#

### CODE REVIEW

**Comment by quincyle** [comment]
Approved

File: blueprints/ember-frost-pods/index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+  /**
+    Installs specified packages at the root level of the application.
+    Triggered by 'ember install <addon name>'.
+    @returns {Promise} package names and versions
+  */
+  afterInstall: function () {
+    return this.addAddonsToProject({
+      packages: [
+        {name: 'ember-frost-core', target: '>=0.0.14 <2.0.0'}
```
Comment (vesper2000) [inline blueprints/ember-frost-pods/index.js]: This version should be increased.  core is now at 0.6.5 (last time I checked)

File: package.json [hunk 2]
```diff
@@ -34,28 +35,31 @@
...
   "dependencies": {
+
     "ember-cli-babel": "^5.1.5",
     "ember-computed-decorators": "^0.2.2",
     "ember-cli-htmlbars": "^1.0.1",
     "ember-cli-sass": "^5.2.0",
-    "ember-frost-theme": "^1.0.1",
-    "liquid-fire": "^0.22.0"
+    "liquid-fire": "^0.22.0",
+    "ember-frost-core": "^0.0.13",
```
Comment (vesper2000) [inline package.json]: Remove the ember-frost addons from here.

File: package.json [hunk 3]
```diff
@@ -34,28 +35,31 @@
...
     "ember-cli-mirage": "~0.1.11",
     "ember-cli-mocha": "^0.10.0",
     "ember-cli-notifications": "^3.2.0",
     "ember-lodash": "~0.0.6",
     "ember-truth-helpers": "^1.2.0",
-    "eslint": "^2.1.0",
-    "eslint-config-frost-standard": "^1.0.0",
+    "eslint": "^2.8.0",
+    "eslint-config-frost-standard": "^2.0.0",
     "ember-frost-css-core": "^1.0.0"
```
Comment (vesper2000) [inline package.json]: This version should match the version in index.js in blueprints.

**Comment by vesper2000** [comment]
Please fix the issues reported.

**Pull request merged**